#! /bin/sh /usr/share/dpatch/dpatch-run
## 10-dist_conf.dpatch by Peter Eisentraut <petere@debian.org>
##
## DP: Adjusts some paths

@DPATCH@
diff -urNad kolabd-2.1.0-20070510.dfsg~/dist_conf/debian kolabd-2.1.0-20070510.dfsg/dist_conf/debian
--- kolabd-2.1.0-20070510.dfsg~/dist_conf/debian	2006-03-09 22:52:02.000000000 +0100
+++ kolabd-2.1.0-20070510.dfsg/dist_conf/debian	2007-07-27 14:07:47.000000000 +0200
@@ -1,14 +1,15 @@
-kolab_usr=root
-kolab_grp=root
-kolab_musr=root
-kolab_mgrp=root
+kolab_usr=kolab
+kolab_grp=kolab
+kolab_musr=kolab
+kolab_mgrp=kolab
 kolab_uid=1002
 kolab_gid=1002
-restricted_kolab_usr=root
-restricted_kolab_grp=root
+restricted_kolab_usr=kolab
+restricted_kolab_grp=kolab
 kolab_rusr=${restricted_kolab_usr}
 kolab_rgrp=${restricted_kolab_grp}
 kolab_susr=root
+pki_grp=${kolab_rgrp}
 
 BASHELL=/bin/bash
 PERL=${bindir}/perl
@@ -25,13 +26,13 @@
 amavisd_templatedir=${sysconfdir}/amavis/conf.d
 amavisd_logfile=${localstatedir}/log/amavis.log
 amavisd_home=${localstatedir}/lib//amavis
-amavisd_rusr=kolab-r
+amavisd_rusr=kolab
 amavisd_usr=kolab
-amavisd_grp=kolab-r
+amavisd_grp=kolab
 
 fsl_logfile=${localstatedir}/log/fsl
 
-sysrundir=${localstatedir}/run/amavis
+sysrundir=${localstatedir}/run
 
 clamav_confdir=${sysconfdir}/clamav
 clamav_socket=${localstatedir}/clamav/clamd.sock
@@ -52,9 +53,10 @@
 emailserver_socket=${libdir}/postfix/lmtp
 emailserver_localstatedir=${localstatedir}/spool/postfix
 emailserver_logfile=${localstatedir}/log/mail.log
-emailserver_usr=${kolab_usr}
+emailserver_usr=root
+emailserver_tables_usr=root
 emailserver_musr=kolab
-emailserver_grp=${restricted_kolab_grp}
+emailserver_grp=root
 
 ftpserver_scoreboardfile=${localstatedir}/proftpd/score
 ftpserver_logfile=${localstatedir}/log/proftpd.log
@@ -69,34 +71,35 @@
 
 imap_confdir=${sysconfdir}
 imap_confperm=0640
-imap_usr=cyrus
-imap_grp=sasl
+imap_usr=root
+imap_grp=root
 imap_masterlogfile=${localstatedir}/log/mail.log
 imap_misclogfile=${localstatedir}/log/mail.log
 imap_statedir=${localstatedir}/lib/cyrus
 imap_spool=${localstatedir}/spool/cyrus/mail
 imap_sievedir=${localstatedir}/spool/sieve
 imap_lmtp=${localstatedir}/run/cyrus/socket/lmtp
+imap_notify_socket=${localstatedir}/run/cyrus/socket/notify
 
 kolab_statedir=${localstatedir}/kolab
 kolab_logdir=${localstatedir}/log/kolab
 kolab_pidfile=${sysrundir}/kolab.pid
 kolab_mailboxuiddb=${kolab_statedir}/mailbox-uidcache.db
-kolab_scriptsdir=$(sysconfdir)/kolab
-kolab_php_smarty_prefix=Smarty
+kolab_scriptsdir=${sbindir}
+kolab_php_smarty_prefix=smarty/libs
 
 ldapserver_statedir=${localstatedir}/lib/ldap
 ldapserver_dir=${localstatedir}/lib/ldap
-ldapserver_replogfile=${localstatedir}/log/ldap-replicalog
-ldapserver_argsfile=${sysrundir}/slapd.args
+ldapserver_replogfile=${ldapserver_statedir}/replog
+ldapserver_argsfile=${sysrundir}/slapd/slapd.args
 ldapserver_logfile=${localstatedir}/log/syslog
-ldapserver_pidfile=${sysrundir}/slapd.pid
-ldapserverslurpd_pidfile=${sysrundir}/slurpd.pid
+ldapserver_pidfile=${sysrundir}/slapd/slapd.pid
+ldapserverslurpd_pidfile=${sysrundir}/slapd/slurpd.pid
 ldapserver_schemadir=${sysconfdir}/ldap/schema
 ldapserver_confdir=${sysconfdir}/ldap
-ldapserver_usr=ldap
-ldapserver_grp=ldap
-ldapserver_rusr=ldap
+ldapserver_usr=openldap
+ldapserver_grp=openldap
+ldapserver_rusr=openldap
 
 webserver_document_root=/var/www
 webserver_cgidir=${webserver_document_root}/cgi-bin
@@ -114,28 +117,37 @@
 webserver_sslmutex=${sysrundir}/ssl_mutex
 webserver_web_prefix=/kolab
 webserver_musr=kolab
-webserver_usr=wwwrun
-webserver_grp=www
+webserver_usr=www-data
+webserver_grp=www-data
 
+phpini_dir=${sysconfdir}/kolab
 phplibdir=${datadir}/php
 phplibdir2=${datadir}/php
 phppeardir=${phplibdir}/PEAR
 
-sasl_authdconffile=${sysconfdir}/sasl/____
-sasl_smtpconffile=${libdir}/sasl2/smtpd.conf
+sasl_authdconffile=${sysconfdir}/saslauthd.conf
+sasl_smtpconffile=/etc/postfix/sasl/smtpd.conf
 sasl_logfile=${localstatedir}/log/saslauthd.log
 
 resmgr_logfile=${localstatedir}/log/resmgr.log
-resmgr_confdir=${sysconfdir}/resmgr
+resmgr_confdir=${sysconfdir}/kolab
 resmgr_filterdir=${localstatedir}/resmgr/filter
-resmgr_scriptsdir=${resmgr_confdir}
-resmgr_usr=kolab
-resmgr_grp=kolab-n
+resmgr_scriptsdir=/usr/share/kolab-resource-handlers
+resmgr_conffile_usr=root
+resmgr_conffile_grp=www-data
+resmgr_usr=root
+resmgr_grp=www-data
 
 fbview_logfile=${localstatedir}/resmgr/fbview.log
 freebusy_logfile=${localstatedir}/resmgr/freebusy.log
 freebusy_cachedir=${webserver_document_root}${webserver_web_prefix}/freebusy/cache
+freebusy_usr=root
+freebusy_grp=www-data
 
 graveyard_uidcache=${localstatedir}/kolab/graveyard_uidcache.db
 graveyard_tscache=${localstatedir}/kolab/graveyard_tscache.db
 
+backupdir=${localstatedir}/backups/kolab
+
+kolab_smtpdpolicy_conffile_usr=${kolab_musr}
+kolab_smtpdpolicy_conffile_grp=${kolab_grp}
